Xamarin.Forms.Xaml.UnitTests/DesignTimeLoaderTests.cs

```diff
@@ -550,6 +550,25 @@ public void IgnoreConverterException()
 			Assert.DoesNotThrow(() => XamlLoader.Create(xaml, true));
 			Assert.That(exceptions.Count, Is.EqualTo(1));
 		}
+
+		[Test]
+		public void IgnoreMarkupExtensionException()
+		{
+			// The user scenario is a user type with a static member referenced in an x:Static markup extension. The
+			// previewer replaces the user type, so the static member does not exist. However we should handle general
+			// cases where x:Static and other markup extensions fail.
```

Comment:
actually, in this case, your x:Static syntax is wrong, but whatever, the point is having a markup that throws...

Reply:
Yes, I know... but either way it ultimately throws in the same method, so it didn't seem important whether I had `{x:Static Foo}` or `{x:Static Foo.Bar}`

Reply:
Happy to include both 😁